packages/bitski-provider/src/constants.ts

```diff
@@ -56,26 +56,26 @@ export const SUPPORTED_CHAIN_IDS = [
   59, 60, 61, 62, 63, 64, 65, 66, 67, 68, 69, 71, 74, 76, 77, 78, 79, 80, 81, 82, 83, 85, 86, 87,
   88, 89, 90, 91, 92, 93, 95, 96, 97, 99, 100, 101, 102, 105, 106, 107, 108, 110, 111, 122, 123,
   124, 125, 126, 127, 128, 137, 141, 142, 162, 163, 168, 170, 172, 186, 188, 189, 199, 200, 211,
-  218, 222, 225, 226, 246, 250, 256, 258, 262, 269, 288, 300, 321, 322, 333, 335, 336, 338, 361,
-  363, 364, 365, 369, 385, 420, 499, 512, 513, 555, 558, 588, 595, 600, 666, 686, 700, 707, 708,
-  721, 777, 787, 788, 803, 820, 821, 880, 888, 900, 901, 902, 903, 940, 941, 942, 977, 998, 999,
-  1001, 1007, 1008, 1010, 1012, 1022, 1023, 1024, 1028, 1030, 1088, 1139, 1140, 1197, 1201, 1202,
-  1213, 1214, 1280, 1284, 1285, 1286, 1287, 1288, 1337, 1618, 1620, 1657, 1688, 1856, 1898, 1984,
-  1987, 2001, 2020, 2021, 2022, 2025, 2100, 2101, 2152, 2153, 2213, 2221, 2559, 3000, 3001, 3331,
-  3333, 3334, 3400, 3500, 3690, 3966, 3967, 4002, 4102, 4689, 4690, 4918, 5197, 5315, 5700, 5777,
-  5851, 5869, 6626, 7341, 7878, 8000, 8001, 8029, 8080, 8217, 8285, 8723, 8724, 8888, 8995, 9000,
-  9001, 9100, 9527, 9999, 10000, 10001, 10101, 10823, 11111, 11437, 12051, 12052, 13381, 16000,
-  16001, 19845, 21337, 21816, 24484, 24734, 31102, 31337, 32659, 39797, 42069, 42161, 42220, 42261,
-  42262, 43110, 43113, 43114, 44787, 45000, 47805, 49797, 53935, 55555, 55556, 60000, 60001, 60002,
-  60103, 62320, 63000, 63001, 69420, 70000, 70001, 70002, 70103, 71393, 73799, 78110, 80001, 99998,
-  99999, 100000, 100001, 100002, 100003, 100004, 100005, 100006, 100007, 100008, 108801, 110000,
-  110001, 110002, 110003, 110004, 110005, 110006, 110007, 110008, 200101, 200625, 201018, 201030,
-  210425, 234666, 246529, 246785, 281121, 333888, 333999, 421611, 444900, 666666, 888888, 955305,
-  1313114, 1313500, 1337702, 2203181, 7762959, 11155111, 13371337, 18289463, 20181205, 28945486,
-  35855456, 61717561, 99415706, 192837465, 245022926, 245022934, 245022940, 311752642, 356256156,
-  486217935, 1122334455, 1313161554, 1313161555, 1313161556, 1666600000, 1666600001, 1666600002,
-  1666600003, 1666700000, 1666700001, 1666700002, 1666700003, 2021121117, 3125659152, 4216137055,
-  11297108099, 11297108109, 197710212030, 197710212031, 6022140761023, 868455272153094,
+  218, 222, 225, 226, 246, 250, 256, 258, 262, 269, 280, 288, 300, 321, 322, 324, 333, 335, 336,
```

Comment:
should probably just make this as a static json resource that can be shared across bitski

Reply:
like a `bitski-chainlist.json`